<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushSoft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Soft Key Scenario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAck">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>task Finished</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushManDown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Man Down</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Task:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentTask">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentScenario">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Scenario:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Messages...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="signalLevel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="caller">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Caller ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="smsBody">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushSoft</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>queueSoft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushFire</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>queueFire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushManDown</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>queueManDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>queueSoft()</slot>
  <slot>queueFire()</slot>
  <slot>queueManDown()</slot>
  <slot>simulateEndOfTask()</slot>
  <slot>pushAckPressed()</slot>
 </slots>
</ui>
